cli/pack/common.go

```diff
@@ -266,6 +266,18 @@ func copyBinaries(bundleEnvPath string, packCtx *PackCtx, cmdCtx *cmdcontext.Cmd
 	if err := util.CopyFileDeep(ttExecutable, util.JoinPaths(pkgBin, "tt")); err != nil {
 		return fmt.Errorf("failed copying tt: %s", err)
 	}
+
+	// Copy tcm.
+	if packCtx.WithBinaries {
```

Comment:
Please, add test cases into `Test_prepareBundle` in the `common_test.go`.

Reply:
Done